airgun/entities/cloud_insights.py

```diff
@@ -51,21 +51,21 @@ def update(self, values):
 
 @navigator.register(CloudInsightsEntity, 'Token')
 class SaveCloudTokenView(NavigateStep):
-    """Navigate to main Red Hat Insights page"""
+    """Navigate to main Red Hat Lightspeed page"""
```

Comment:
This file needs more updates for `s/Insights/Lightspeed`, could you update those too?

Reply:
I am only updated the doc string for any class or method that actually needed a Insights to light speed change. Changed every doc string and var name across robottelo will be another task once we finally update all the light speed changes in satellite. This feature was just a quick branding change